pylon-core/src/main/kotlin/io/github/pylonmc/pylon/core/util/gui/ProgressItem.kt

```diff
@@ -57,6 +65,11 @@ abstract class ProgressItem @JvmOverloads constructor(
         return builder
     }
 
+    /**
+     * This can be used to modify the item being displayed.
```

Comment:
```suggestion
     * This is be used to modify the item being displayed.
```
I couldn't sworn I said this exact thing before...?

Reply:
This is be used?

Reply:
Fixed, ty